actionpack/lib/action_dispatch/routing.rb

```diff
@@ -250,7 +250,9 @@ module Routing
 
     autoload :Mapper
     autoload :RouteSet
-    autoload :RoutesProxy
+    eager_autoload do
```

Comment:
Are we calling `Routing.eager_load!` anywhere?

Reply:
No you're right, maybe we should automatically propagate these calls though.